src/utils/private.ts

```diff
@@ -161,6 +161,13 @@ export const getWideConfig = async (context: Context) => {
     openAIKey: mergedConfigData.openAIKey,
     openAITokenLimit: mergedConfigData.openAITokenLimit,
     staleBountyTime: mergedConfigData.staleBountyTime,
+    startCommandEnabled: mergedConfigData.startCommandEnabled,
+    defaultTimeRangeForMaxIssue: mergedConfigData.defaultTimeRangeForMaxIssue,
+    defaultTimeRangeForMaxIssueEnabled: mergedConfigData.defaultTimeRangeForMaxIssueEnabled,
+    defaultPermitBaseUrl: mergedConfigData.defaultPermitBaseUrl,
+    defaultBotDelay: mergedConfigData.defaultBotDelay,
+    defaultFollowUpTime: mergedConfigData.defaultFollowUpTime,
+    defaultDisqualifyTime: mergedConfigData.defaultDisqualifyTime,
```

Comment:
```suggestion
    disqualifyTime: mergedConfigData.disqualifyTime,
```
they shouldn't have `default` in front of them